### DISCUSSION TITLE
OTA Commands - image block request

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy#1073] [Q&A] [answered]
I had a look into the zigpy code and especially in the zigpy/zcl/general.py on how the OTA image block request is handle, and I might have badly read, but I don't see the management of the response_spacing parameter in the command .

Is that on purpose ?

**Comment by puddly** [comment, marked as answer]
The OTA code has not been modified since my proof of concept a few years ago. It's due for a rewrite. If you have any suggestions, I'd appreciate them.

That being said, I don't think I've ever encountered the `image_page` command, only `image_block`. Where are you seeing it used?

**Reply by pipiche38** [reply]
Legrand remote is using image page request with page size of 256 bytes , block size of 64 bytes and the delay between block is 200ms
